docs/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "printWidth": 150,
```

Comment:
Why don't you make it 100 here and omit the second prettierrc?

Reply:
The outer config is used for all the actual code (e.g. everything in `docs/src/`) and I think that should be the same as the code in all our other packages, where it is also 150.